cli/commands/configure-device.ts

```diff
@@ -24,7 +29,7 @@ export const configureDeviceCommand = ({
 	devicesIndexName: string
 	stackName: string
 }): CommandDefinition => ({
-	command: 'configure-device <fingerprint>',
+	command: 'configure-device <account> <fingerprint>',
```

Comment:
This should not be neccessary. We know the association between a device and an account. Fingerprints are globally unique.